person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [ ] According to the project requirement, kindly implement a method to create a new rental using the `Rentals` class.

 ```
def rent_book(book, date)
    Rental.new(date, book, self)
  end
```